app/controllers/openid_application_controller.rb

```diff
@@ -139,13 +138,14 @@ def fetch_aud
   def fetch_smart_launch_context
     response = RestClient.get(Settings.oidc.smart_launch_url,
                               { Authorization: 'Bearer ' + token.token_string })
-    unless response.nil? || response.code != 200
+    raise error_klass('Invalid launch context') if response.nil?
```

Comment:
This is more of a me learning ruby comment-Is this common or the expected style in ruby, particularly when the thing that would usually come after **if** is a one liner? I would normally say for consistency to change it to a 
if _____
    then ____
but I have a feeling this is not uncommon for this situation.

Reply:
yah it's not uncommon in ruby. I think the linter would have picked it up if it was also.